pytorch_lightning/trainer/trainer.py

```diff
@@ -270,8 +271,15 @@ def __init__(
             num_sanity_val_steps: Sanity check runs n validation batches before starting the training routine.
                 Set it to `-1` to run all batches in all validation dataloaders.
 
+            reload_dataloaders_every_n_epochs: Set to a non negative integer to reload dataloaders every n epochs.
+                Default: 0
+
             reload_dataloaders_every_epoch: Set to True to reload dataloaders every epoch.
 
+            .. warning::
+               ``reload_dataloaders_every_epoch`` has been deprecated in v1.4 and will be removed in v1.6.
+               Please use ``reload_dataloaders_every_n_epochs``.
```

Comment:
check the rendered docs, this will not display correctly. 
needs an indentation.